tools/python3/bench.py

```
@@ -0,0 +1,273 @@
+
+def print_summary(filepath=None, root_id=None):
+    if filepath:
+        return handle_input(filepath)
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input", default=None, help="input")
+    args = parser.parse_args()
+    if args.input:
+        return handle_input(args.input)
+
+    for el in Path(".phare_bench").iterdir():
```

Ruff (0.15.14)

[warning] Loop control variable `el` not used within loop body

(B007)